decidim-bulletin_board-js/src/trustee/identification_keys.js

```diff
@@ -0,0 +1,256 @@
+/**
+ * IdentificationKeys component.
+ */
+export class IdentificationKeys {
+  constructor(trusteeUniqueId, storedPublicKey) {
+    this.format = "jwk";
+    this.algorithm = {
+      name: "RSASSA-PKCS1-v1_5",
+      modulusLength: 4096,
```

Comment:
It would be great to share this constant with the Ruby code. I think is very hard to do that, but then we should define a constant for that.

Reply:
I am going to move it to a constant, and we can think about sharing this with Ruby. Maybe with some kind of JSON configuration file or so.